core/settings/qPrefAnimations.h

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-2.0
+#ifndef QPREFANIMATIONS_H
+#define QPREFANIMATIONS_H
+
+#include <QObject>
+#include <QSettings>
+
+class qPrefAnimations : public QObject {
+	Q_OBJECT
+	Q_PROPERTY(int animation_speed READ animation_speed WRITE set_animation_speed NOTIFY animation_speed_changed);
+
+public:
+	qPrefAnimations(QObject *parent = NULL);
+	static qPrefAnimations *instance();
+
+	// Load/Sync local settings (disk) and struct preference
+	void loadSync(bool doSync);
+	void load() { loadSync(false); };
```

Comment:
Pedantic mode: In C (and by inheritance in C++), function definitions don't end in ';'.
What you are writing is the equivalent of
```
struct {
	int a;;
};
```
Sure - it's valid code but also kind of ugly. ;)

Reply:
Pedantic mode is surely ok, but I find it interesting you comment it here and not before (all my PR with class definitions have this).

It is not a big deal for me, except for the monster PR it calls for (to change C++ header files which uses this practice, a quick look on other header files shows this seems to be common practice).

I like to use ; to ensure the statement is terminated and do not find it ugly in contrary :-)

Reply:
Could not resist a cross check, please have a look at your own #1486 core/imagedownloader.h  and core/videoframeextractor.h why is it not kind of ugly there :-)

Also if I change the function defintion to:
      void loadSync(bool doSync)
      void load() { loadSync(false); }
I get clang compiler errors

If I change the function call (you wrote definition):
      void load() { loadSync(false)  };
I get clang compiler errors

So what ; do you find kind of ugly ?